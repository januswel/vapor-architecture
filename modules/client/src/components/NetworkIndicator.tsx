import * as React from 'react'
import { ActivityIndicator, StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  container: {
    position: 'absolute',
    bottom: 0,
    right: 0,
  },
})

export default () => <ActivityIndicator style={styles.container} size="large" />
