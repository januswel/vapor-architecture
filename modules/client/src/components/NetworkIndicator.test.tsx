import * as React from 'react'
import * as renderer from 'react-test-renderer'

import NetworkIndicator from './NetworkIndicator'

describe('NetworkIndicator', () => {
  it('renders indicator', () => {
    const component = renderer.create(<NetworkIndicator />)
    expect(component).toMatchSnapshot()
  })
})
