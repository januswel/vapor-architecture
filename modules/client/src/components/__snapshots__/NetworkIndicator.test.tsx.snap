// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetworkIndicator renders indicator 1`] = `
<ActivityIndicator
  animating={true}
  color="#999999"
  hidesWhenStopped={true}
  size="large"
  style={
    Object {
      "bottom": 0,
      "position": "absolute",
      "right": 0,
    }
  }
/>
`;
